test/e2e/auth/audit.go

```diff
@@ -54,23 +54,609 @@ var (
 	patch, _     = json.Marshal(jsonpatch.Patch{})
 )
 
-var _ = SIGDescribe("Advanced Audit", func() {
+// TODO: Get rid of [DisabledForLargeClusters] when feature request #53455 is ready.
```

Comment:
#53455 seems to be obsolete?

Reply:
I think we still want it, though it might make more sense to rewrite this test using dynamic audit features, once that graduates.